CMakeLists.txt

```diff
@@ -220,6 +219,16 @@ target_compile_definitions      (libtesseract
 )
 set_target_properties           (libtesseract PROPERTIES WINDOWS_EXPORT_ALL_SYMBOLS True)
 endif()
+if (Tesseract_USE_OPENCL)
+    target_compile_definitions  (libtesseract PRIVATE USE_OPENCL=1)
+    if (APPLE)
+        # Inbuilt to NDK
+    else()
+        hunter_add_package(OpenCL)
+    endif()
+    find_package(OpenCL CONFIG REQUIRED)
+    target_link_libraries       (libtesseract PUBLIC OpenCL::OpenCL)
```

Comment:
`find_package(OpenCL REQUIRED)`/`find_package(OpenCL CONFIG REQUIRED)` should be called in `*Config.cmake` in this case.